docs-ref-services/storage.md

```diff
@@ -1,62 +1,41 @@
+|**Storage File Datalake**|[@azure/storage-file-datalake](https://www.npmjs.com/package/@azure/storage-file-datalake)|[storage-file-datalake-typescript-examples](https://docs.microsoft.com/samples/azure/azure-sdk-for-js/storage-file-datalake-typescript/)<br> [storage-file-datalake-javascript-examples](https://docs.microsoft.com/samples/azure/azure-sdk-for-js/storage-file-datalake-javascript/)|[Readme - Azure Storage File Datalake](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/storage/storage-file-datalake/README.md)|
+|**Storage Queue**|[@azure/storage-queue](https://www.npmjs.com/package/@azure/storage-queue)|[storage-queue-typescript-examples](https://docs.microsoft.com/samples/azure/azure-sdk-for-js/storage-queue-typescript/)<br> [storage-queue-javascript-examples](https://docs.microsoft.com/samples/azure/azure-sdk-for-js/storage-queue-javascript/)|Send and receive messages between cloud-connected applications with <br>[Azure Storage Queue](https://docs.microsoft.com/azure/storage/queues/storage-quickstart-queues-nodejs)|
+|**Storage Table**|[azure-storage](https://www.npmjs.com/package/azure-storage)<br>(Legacy)| - |Read and write large structured data with [Azure Storage Table](https://docs.microsoft.com/azure/storage/storage-nodejs-how-to-use-table-storage)|
+|||||
 
+Install the npm module with `npm install` followed by the `package-name`. For example,
 ```bash
-npm install azure-storage
+npm install @azure/storage-blob
 ```
+and look at the examples from the links provided in the table above.
 
-### Example
-
-This example create a storage container and uploads a local file `data.txt` to it.
+Read more about the client packages here - [Azure Storage Client Libraries For JavaScript](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/storage/).
 
-```javascript
-const azure = require('azure-storage');
-const blobService = azure.createBlobService(storageConnectionString);
-
-const container = 'taskcontainer';
-const task = 'taskblob';
-const filename = 'data.txt';
-
-blobService.createContainerIfNotExists(container, error => {
-  if (error) return console.log(error);
-  blobService.createBlockBlobFromLocalFile(
-    container,
-    task,
-    filename,
-    (error, result) => {
-      if (error) return console.log(error);
-      console.dir(result, { depth: null, colors: true });
-    }
-  );
-});
-```
+Find more getting started guides at [Browse code samples](https://azure.microsoft.com/resources/samples/)
 
 ## Management Package
```

Comment:
Can we have a 1 line description that tells the user what a "management package" is?

Reply:
That's a very interesting question.

Which one do you like?
> Management packages enable you to create, update, delete, and list resources in your Azure subscription.

or

> Azure storage management package enables you to create, update, delete, and list storage resources in your Azure subscription.

Or anything else you'd suggest?